app/views/users/profile.html.erb

```diff
@@ -268,6 +268,8 @@
       <li><h5><a href = "/tag/activity:*/author/<%= params[:id] %>"><%= pluralize(@count_activities_posted, 'activity posted', plural:'activities posted') %> </a></h5></li>
       <li><h5><a href = "/tag/replication:*/author/<%= params[:id] %>"><%= pluralize(@count_activities_attempted, 'activity attempted', plural:'activities attempted')%></a> </h5></li>
       <li><h5><a href = "/tag/with:<%= @profile_user.username %>"><%= pluralize(Tag.find_nodes_by_type("with:#{@profile_user.username}").count, 'note co-authored', plural: 'notes co-authored')%></a> </h5></li>
+      <li><h5><a href = "/people/<%= params[:id]%>/followers"><%= pluralize(following_path(@user.username).count, 'follower', plural:'followers')%></a></h5></li>
```

Comment:
Hi! Thanks so much for taking this on! Can you help me unpack this line... where is the definition for `following_path()` that should yield the count? I searched for https://github.com/publiclab/plots2/search?q=following_path but didn't get a definition. Could we instead use `followers.size`?

```suggestion
      <li><h5><a href = "/people/<%= params[:id]%>/followers"><%= @user.size %> following</a></h5></li>
```

Reply:
Sorry, but can I ask why did you use`@user.size %> following</a>` rather than `@user.size %> followers</a>` since we are counting the followers?